actions-sample-plugin/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Logs
```

Comment:
We don't need to include the gitignore either

Reply:
I believe this is necessary, otherwise all the node modules will be committed too